.github/workflows/createReleaseBranch.yml

```diff
@@ -67,7 +67,10 @@ jobs:
   slack_notification:
     if: ${{ always() }}
     needs: create_branch
-    uses: ./.github/workflows/slack-notification.yml
+    uses: ./.github/workflows/slackNotification.yml
+    secrets: 
+      SLACK_WEBHOOK_URL: ${{ secrets.VSCODE_SLACK_WEBHOOK }}
+      IDEE_RELEASE_ALERT_SLACK_WEBHOOK: ${{ secrets.IDEE_RELEASE_ALERT_SLACK_WEBHOOK }}
```

Comment:
this was the trick.  It needs to have the secrets passed.  Probably could have also used [inherit](https://github.blog/changelog/2022-05-03-github-actions-simplify-using-secrets-with-reusable-workflows/)

Reply:
yeah me to...will update